fontes/interpretador/interpretador-base.ts

```diff
@@ -561,7 +561,7 @@ export class InterpretadorBase implements InterpretadorInterface {
             if (entidadeChamada instanceof FuncaoPadrao) {
                 try {
                     return entidadeChamada.chamar(
-                        argumentos.map((a) => (a !== null && a.hasOwnProperty('valor') ? a.valor : a)),
+                        argumentos.map((a) => (a !== null && a.valor.hasOwnProperty('valor') ? a.valor.valor : a.valor)),
```

Comment:
nesse caso alguns casos pode ser que nao funcione se o a.valor for um inteiro e vai quebrar, acho que seria ideal algo como 
```js 
argumentos.map((a) => (a !== null && a.valor?.valor || a?.valor))
```

dessa forma ele verifica se existe o valor nas chaves, mesmo que em ambos nao exista em vez de retornar ele, ele ira como undefined, indo dessa forma acredito que todos os testes vao passar

Reply:
Boa sugestão. Modifiquei para algo muito próximo disso.